infection.json

```diff
@@ -0,0 +1,19 @@
+{
+    "$schema": "vendor/infection/infection/resources/schema.json",
+    "source": {
+        "directories": [
+            "src"
+        ]
+    },
+    "logs": {
+        "text": ".infection/infection.log",
+        "summary": ".infection/summary.log",
+        "json": ".infection/infection-log.json",
+        "perMutator": ".infection/per-mutator.md"
+    },
+    "minMsi": 81,
+    "minCoveredMsi": 89,
```

Comment:
These were arbitrary numbers I chose so I could see it fail the CI step then make it pass without having to make loads of changes.

Reply:
Maybe in the future, we could increase it to a number close to the actual results. I think it is OK by now.

Reply:
I think you could move the numbers up, make it red, pick off some mutations and make it green again.